Source/Fuse.Scripting.JavaScript/ThreadWorker.uno

```diff
@@ -140,21 +133,24 @@ namespace Fuse.Scripting.JavaScript
 		}
 
 		/*
-			Throws an exception that was generated on the thread. If there are more than one then the previous
-			ones will be reported and the last one thrown.
+			Dispatches an exception from the JS-thread to the UI-thread.
+		*/
+		void DispatchException(Exception e)
+		{
+			_exceptionQueue.Enqueue(e);
+			UpdateManager.PostAction(CheckAndThrow);
+		}
+
+		/*
+			Throws an exception that was generated on the thread.
 		*/
 		public void CheckAndThrow()
```

Comment:
I think this could be private now. Otherwise LGTM :)

Reply:
Yeah, good call!

Reply:
@cbaggers: Fixed!